components/payment.js

```diff
@@ -131,7 +131,7 @@ export const useWalletPayment = () => {
   const wallet = useWallet()
 
   const waitForWalletPayment = useCallback(async ({ id, bolt11 }, waitFor) => {
-    if (!wallet) {
+    if (!wallet?.enabled) {
```

Comment:
If we didn't have a wallet (e.g. anon), this function would throw

Reply:
Thanks for the fix but I think [1b6de0b](https://github.com/stackernews/stacker.news/pull/1243/commits/1b6de0bb96a0909280eaec926f6473fc646afbb1) is a better fix.

My intention with `useWallet()` (with no name as argument) was supposed to always return an enabled wallet if one exists else null. So we shouldn't need to check if it's enabled if we use `useWallet()`.